lib/game.rb

```diff
@@ -2,8 +2,6 @@ class Game
 
   # take_turns method
 
-  #check_for_winner method
-
   # celebrate_winner
```

Comment:
still need to create this one in the next work session, as well as game.play

Reply:
Yes! And we need to wrap all our helper methods up in the play method